app/views/users/new.html.haml

```diff
@@ -2,13 +2,16 @@
   .row
     .col-sm-10.col-sm-offset-1
       = bootstrap_form_for @user, layout: :horizontal, url: register_path do |f|
+        - if @invitation
+          = hidden_field_tag :invitation_token, @invitation.token
+
         %header
           %h1 Register
         %fieldset
           = f.email_field :email, control_col: 'col-sm-6', label: 'Email Address'
```

Comment:
You can set `control_col: 'col-sm-6'` at form level on line 4. instead of setting the same col at each field.